x-pack/plugins/rule_registry/server/alert_data_client/tests/get.test.ts

```diff
@@ -151,12 +173,12 @@ describe('get()', () => {
         },
       })
     );
-    await alertsClient.get({ id: '1', index: '.alerts-observability-apm' });
+    await alertsClient.get({ id: 'NoxgpHkBqbdrfX07MqXV', index: '.alerts-observability-apm' });
 
     expect(auditLogger.log).toHaveBeenCalledWith({
       error: undefined,
-      event: { action: 'alert_get', category: ['database'], outcome: 'success', type: ['access'] },
-      message: 'User has accessed alert [id=1]',
+      event: { action: 'alert_get', category: ['database'], outcome: 'unknown', type: ['access'] },
```

Comment:
`get` should never have `outcome: 'unknown'`; we know the outcome (we fetched the object and returned it to the user)

Reply:
Ah! Yes thank you. Good catch.

Fixed here: 36838ab2e10ba9b2640665345758373075e0fa44